tpmenc.go

```diff
@@ -0,0 +1,148 @@
+package krypto
+
+import (
+	"crypto"
+	"crypto/rsa"
+	"crypto/sha256"
+	"fmt"
+	"io"
+
+	"github.com/google/go-tpm-tools/client"
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+)
+
+type TpmEncoder struct {
+	publicSigningKey    *rsa.PublicKey
+	publicEncryptionKey *rsa.PublicKey
+	ExternalTpm         io.ReadWriteCloser
+}
+
+func encryptionKey(tpm io.ReadWriteCloser) (tpmutil.Handle, crypto.PublicKey, error) {
+	return tpm2.CreatePrimary(tpm, tpm2.HandleOwner, tpm2.PCRSelection{}, "", "", tpm2.Public{
+		Type:       tpm2.AlgRSA,
+		NameAlg:    tpm2.AlgSHA1,
+		Attributes: tpm2.FlagDecrypt | tpm2.FlagUserWithAuth | tpm2.FlagFixedParent | tpm2.FlagFixedTPM | tpm2.FlagSensitiveDataOrigin,
+		RSAParameters: &tpm2.RSAParams{
+			KeyBits: 2048,
```

Comment:
If the 2048 on line 88 is related to this, maybe this should be a named const

Reply:
done